src/ui_ng/src/app/project/create-project/create-project.component.html

```diff
@@ -23,9 +23,8 @@ <h3 class="modal-title">{{'PROJECT.NEW_PROJECT' | translate}}</h3>
                 <div class="form-group" style="padding-left: 135px;">
                     <label class="col-md-4 form-group-label-override">{{'PROJECT.ACCESS_LEVEL' | translate}}</label>
                     <div class="checkbox-inline">
-                        <input type="checkbox" id="create_project_public" [(ngModel)]="project.public" name="public">
-                        <label for="create_project_public"></label>
-                        <span class="access-level-label">{{ accessLevelDisplayText | translate}}</span>
+                        <input type="checkbox" id="create_project_public" [(ngModel)]="project.public" name="public"><label for="create_project_public"></label>
```

Comment:
Project level policy feature changed the db schema. so public column not exist in project table anymore.
should cherry-pick #3401 first , then replace 'project.public' to 'project.metadata.public'